docs/outlook/autolaunch-store-options.md

```diff
@@ -2,13 +2,13 @@
 title: AppSource listing options for your event-based Outlook add-in
 description: Learn about the AppSource listing options available for your Outlook add-in that implements event-based activation.
 ms.topic: article
-ms.date: 07/11/2022
+ms.date: 09/07/2022
 ms.localizationpriority: medium
 ---
 
 # AppSource listing options for your event-based Outlook add-in
 
-At present, add-ins must be deployed by an organization's admins for end-users to access the event-based feature functionality. We're restricting event-based activation if the end-user acquired the add-in directly from AppSource. For example, if the Contoso add-in includes the `LaunchEvent` extension point with at least one defined `LaunchEvent Type` under the `LaunchEvents` node, the automatic invocation of the add-in only happens if the add-in was installed for the end-user by their organization's admin. Otherwise, the automatic invocation of the add-in is blocked. See the following excerpt from an example add-in manifest.
+At present, add-ins must be deployed by an organization's admins for end-users to access the event-based feature functionality. We're restricting event-based activation if the end-user acquired the add-in directly from [AppSource](https://appsource.microsoft.com). For example, if the Contoso add-in includes the `LaunchEvent` extension point with at least one defined `LaunchEvent Type` under the `LaunchEvents` node, the automatic invocation of the add-in only happens if the add-in was installed for the end-user by their organization's admin. Otherwise, the automatic invocation of the add-in is blocked. See the following excerpt from an example add-in manifest.
 
 ```xml
```

Comment:
It looks like GitHub condensed what appears below because I didn't make any changes to that section. Are you able to expand the additional lines after line 13 by any chance?

Reply:
Ah yes, it was just on the left side of the screen, not the right side I was reviewing! DUH